coriolis/osmorphing/debian.py

```diff
@@ -122,6 +122,17 @@ def set_net_config(self, nics_info, dhcp):
             cfg_name = "%s/coriolis_netplan.yaml" % netplan_base
             self._write_file_sudo(cfg_name, new_cfg)
 
+    def get_installed_packages(self):
+        cmd = "dpkg-query -f '${binary:Package}\n' -W"
+        try:
+            apt_list_output = self._exec_cmd_chroot(
```

Comment:
Also, could you please check whether you could do the following:
```python
apt_list_output = self._exec_cmd_chroot(cmd).decode().splitlines()
```
This way, you only call for `decode()` once.